docs/config.mako

```diff
@@ -35,7 +35,7 @@
 
     hljs_style = "atom-one-dark"
 
-    # Doesn't really do anything, just enables lurn_search
+    # Doesn't really do anything, just enables lunr_search
```

Comment:
You can leave this one out, it's being removed anyways :)

Reply:
oh okay, will do!